scopes/component/graph/ui/dependencies-compare/dependency-compare-node.module.scss

```diff
@@ -0,0 +1,84 @@
+.compNode {
+  max-width: 400px;
```

Comment:
try different componet / file names
could be very long
check what happens when it overflows
and if needed use the ellipsis component